app/test_main.py

```diff
@@ -1,3 +1,79 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_year,dog_year,expected_result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2])
+    ]
+)
+def test_should_equal_expected_result(
+    cat_year: int,
+    dog_year: int,
+    expected_result: list[int, int]
```

Comment:
The type annotation for `expected_result` should be `list` or `List[int]` instead of `list[int, int]`. The latter is not a valid type annotation in Python.